consensus/polybft/consensus_runtime.go

```diff
@@ -20,9 +21,10 @@ import (
 )
 
 const (
-	eventsBufferSize   = 10
-	stateFileName      = "consensusState.db"
-	uptimeLookbackSize = 2 // number of blocks to calculate uptime from the previous epoch
+	eventsBufferSize       = 10
+	stateFileName          = "consensusState.db"
+	uptimeLookbackSize     = 2                // number of blocks to calculate uptime from the previous epoch
+	checkpointTimeInterval = 30 * time.Minute // frequency at which sending checkpoints are sent to the rootchain
```

Comment:
I would move this `checkpointTimeInterval` to `checkpointManager`, so that all checkpoint related configuration resides there.

Reply:
Fixed [6a2a52b](https://github.com/0xPolygon/polygon-edge/pull/861/commits/6a2a52ba4577cd8c48852e9273e77ae3fc5492fe)